### PR TITLE
Req approvals

### PR DESCRIPTION
**Pull request by self-maurya**

### CODE REVIEW

**Review by dplore — DISMISSED** [review]

**Comment by dplore** [comment]
@self-maurya can you test this in another branch where there are two code owner groups so we can confirm that this github action will require one approver from each group?

File: .github/workflows/required_approvals.yml
Comment (xw-g) [inline .github/workflows/required_approvals.yml:24]: say I have 2 user-groups for an entry in the OWNER file. Does it mean 1 LGTM from each group, or 2 LGTM from the same group is ok too? Can you help add some comment here for future readers? Thanks.

File: .github/workflows/required_approvals.yml
Comment (self-maurya) [reply .github/workflows/required_approvals.yml:1]: done

**Comment by self-maurya** [comment]
> @self-maurya can you test this in another branch where there are two code owner groups so we can confirm that this github action will require one approver from each group?

I think we have to merge this PR. After that I'll enable codeowners approvals from repo branch setting, then on a dummy PR we can do the testing.

**Comment by dplore** [comment]
> > @self-maurya can you test this in another branch where there are two code owner groups so we can confirm that this github action will require one approver from each group?
> 
> I think we have to merge this PR. After that I'll enable codeowners approvals from repo branch setting, then on a dummy PR we can do the testing.

You can create dummy PR's now.  The way the github actions work is the run even in the PR you have created.

For example, this very PR runs the "required_approvals" github action which only exists in the PR (not the main branch).


Link to gist: https://github.com/openconfig/featureprofiles/actions/runs/11721349696/job/32648581511?pr=3563

**Review by dplore — APPROVED** [review]
approving as a test of the req-approvals CI check

**Review by rohit-rp — APPROVED** [review]

**Pull request merged**
